src/js/utils/DOM.js

```diff
@@ -85,10 +85,18 @@ export const getBodyChildElements = () => {
   return children;
 };
 
-export const getNewContainer = (rootNode = document.body) => {
+export const getNewContainer = (
+  rootNode = document.body,
```

Comment:
since we're calling the new argument `targetChildPosition`, I think we should rename `rootNode` to `target` here.

Reply:
Done.